diff --git a/config/btop/btop.conf b/config/btop/btop.conf
index c843983..e41208e 100644
--- a/config/btop/btop.conf
+++ b/config/btop/btop.conf
@@ -2,7 +2,7 @@
 
 #* Name of a btop++/bpytop/bashtop formatted ".theme" file, "Default" and "TTY" for builtin themes.
 #* Themes should be placed in "../share/btop/themes" relative to binary or "$HOME/.config/btop/themes"
-color_theme = "Default"
+color_theme = "tokyo-night"
 
 #* If the theme set background should be shown, set to False if you want terminal background transparency.
 theme_background = True
@@ -22,7 +22,7 @@ presets = "cpu:1:default,proc:0:default cpu:0:default,mem:0:default,net:0:defaul
 
 #* Set to True to enable "h,j,k,l,g,G" keys for directional control in lists.
 #* Conflicting keys for h:"help" and k:"kill" is accessible while holding shift.
-vim_keys = False
+vim_keys = True
 
 #* Rounded corners on boxes, is ignored if TTY mode is ON.
 rounded_corners = True
@@ -32,7 +32,7 @@ rounded_corners = True
 #* "block" has half the resolution of braille but uses more common characters.
 #* "tty" uses only 3 different symbols but will work with most fonts and should work in a real TTY.
 #* Note that "tty" only has half the horizontal resolution of the other two, so will show a shorter historical view.
-graph_symbol = "braille"
+graph_symbol = "tty"
 
 # Graph symbol to use for graphs in cpu box, "default", "braille", "block" or "tty".
 graph_symbol_cpu = "default"
@@ -47,10 +47,10 @@ graph_symbol_net = "default"
 graph_symbol_proc = "default"
 
 #* Manually set which boxes to show. Available values are "cpu mem net proc", separate values with whitespace.
-shown_boxes = "cpu mem net proc"
+shown_boxes = "cpu proc"
 
 #* Update time in milliseconds, recommended 2000 ms or above for better sample times for graphs.
-update_ms = 2000
+update_ms = 500
 
 #* Processes sorting, "pid" "program" "arguments" "threads" "user" "memory" "cpu lazy" "cpu direct",
 #* "cpu lazy" sorts top process over time (easier to follow), "cpu direct" updates top process directly.
@@ -98,7 +98,7 @@ cpu_invert_lower = True
 cpu_single_graph = False
 
 #* Show cpu box at bottom of screen instead of top.
-cpu_bottom = False
+cpu_bottom = True
 
 #* Shows the system uptime in the CPU box.
 show_uptime = True
@@ -203,4 +203,4 @@ selected_battery = "Auto"
 
 #* Set loglevel for "~/.config/btop/btop.log" levels are: "ERROR" "WARNING" "INFO" "DEBUG".
 #* The level set includes all lower levels, i.e. "DEBUG" will show all logging info.
-log_level = "WARNING"
\ No newline at end of file
+log_level = "INFO"
